include/HttpResponse.hpp

```diff
@@ -16,21 +16,24 @@ class HttpResponse
 			const std::pair<unsigned int, std::string> &status,
 			HttpRequest *request);
 
-		void	setBody(const std::string body);
-		void	setStatus(const std::pair<unsigned int, std::string> &status);
-		bool	contentTypeSet(std::string resourcePath, std::string contentTypeToFind);
+		void						setBody(const std::string body);
```

Comment:
I have been removing these tabulations, its super annoying and unnecessary to keep adding all these tabs :P

Reply:
Ah, i have been adding them in, cause I think the code looks cleaner and is easier to see when its tabulated evenly.

Reply:
But its like 15 fucking tabulators  :(:(:(:(:(:( Id suggest that things that dont belong together would be separated with a new line instead of grouped with tabs

Reply:
ok